packages/@okta/vuepress-site/guides/sign-into-spa/section09/react/reqautheverything.md

```diff
@@ -1 +1,20 @@
-placeholder
\ No newline at end of file
+We provide a `SecureRoute` component which extends the `Route` component to require authentication. Use `SecureRoute` on all routes to require authentication before accessing the entire application.
```

Comment:
We provide a `SecureRoute` component that extends the `Route` component to require authentication. Use `SecureRoute` on all routes to require authentication before accessing the entire application.